corehq/toggles/__init__.py

```diff
@@ -1429,14 +1429,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
-SEND_UCR_REBUILD_INFO = StaticToggle(
```

Comment:
Should I add a migration to delete the toggle data as well? I see no problem keeping the data as it's small enough and allows rolling back for whatever reason.